docs/MultiTableDatabase-batchsource.md

```diff
@@ -51,7 +51,7 @@ By default reads all tables matching the tableNamePattern.
 **blackList**: Used in conjunction with tableNamePattern, this configuration specifies the tables to be skipped.
 By default the black list is empty which means no tables will be skipped.
 
-**numSplit**: Number of splits per table. By default is 1
+**splitsPerTable**: The number of splits per table. By default is 1
```

Comment:
as mentioned before, please change these properties to use the UI label. 'Splits Per Table' 'Where Clause', etc.

Reply:
done